Reference/Routing/Request-Pipeline/IContentFinder.md

```diff
@@ -72,6 +72,13 @@ namespace My.Website
 }
 
 ```
+:::note
+In Umbraco V7 there existed an IContentFinder that would find content and display it with an 'alternative template' via a convention - (to avoid the ugly ?alttemplate=blogfullstory appearing on the querystring of the url when using the alternative template mechanism). Instead the Url could follow the convention of /urltocontent/altemplatealias 
```

Comment:
```suggestion
In Umbraco7 there existed an IContentFinder that would find content and display it with an 'alternative template' via a convention. This could be to avoid the ugly `?alttemplate=blogfullstory` appearing on the querystring of the url when using the alternative template mechanism. Instead the Url could follow the convention of `/urltocontent/altemplatealias`. 
```